db/migrate/20151110082345_add_metadata_to_spotlight_resource.rb

```diff
@@ -1,5 +1,12 @@
 class AddMetadataToSpotlightResource < ActiveRecord::Migration
   def up
-    add_column :spotlight_resources, :metadata, :blob
+    # postgresql does not have a blob type
+    conn_type = connection.adapter_name.downcase.to_sym
+    field_type = (conn_type == :postgresql) ? :bytea : :blob
+    add_column :spotlight_resources, :metadata, field_type
```

Comment:
That [seems right](http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/TableDefinition.html#method-i-column). I can give it another go tomorrow and see if it works.